src/ReservationItem.php

```diff
@@ -401,26 +402,13 @@ public function showForm($ID, array $options = [])
                 }
             }
 
-            echo "<div class='center'><form method='post' name=form action='" . $this->getFormURL() . "'>";
-            echo "<input type='hidden' name='id' value='$ID'>";
-            echo "<table class='tab_cadre'>";
-            echo "<tr><th colspan='2'>" . __s('Modify the comment') . "</th></tr>";
-
-           // Ajouter le nom du materiel
-            echo "<tr class='tab_bg_1'><td>" . _n('Item', 'Items', 1) . "</td>";
-            echo "<td class='b'>" . sprintf(__('%1$s - %2$s'), $type, $name) . "</td></tr>\n";
-
-            echo "<tr class='tab_bg_1'><td>" . __('Comments') . "</td>";
-            echo "<td><textarea name='comment' cols='30' rows='10' >" . $r->fields["comment"];
-            echo "</textarea></td></tr>\n";
-
-            echo "<tr class='tab_bg_2'><td colspan='2' class='top center'>";
-            echo "<input type='submit' name='update' value=\"" . _sx('button', 'Save') . "\" class='btn btn-primary'>";
-            echo "</td></tr>\n";
-
-            echo "</table>";
-            Html::closeForm();
-            echo "</div>";
+            $options['candel'] = false;
+            $this->initForm($ID, $options);
```

Comment:
This page is really hard to find (not your fault).

Display should be reworked. Previous state lacked polish, but it's worse now.

Previous state:
![image](https://github.com/glpi-project/glpi/assets/33253653/64416888-8a40-47d1-b38d-ba1cbd35ae32)

Now:
![image](https://github.com/glpi-project/glpi/assets/33253653/b4f88252-1cce-4793-bc53-0060aab689e3)